pytorch_blade/tests/torchscript/view_likes.graph

```diff
@@ -0,0 +1,40 @@
+// RUN: shape_analysis_tool -f %s | FileCheck %s
+
+// CHECK-LABEL: graph
+graph(%p1 : Half(16384, 768)):
+  %cst768: int = prim::Constant[value=768]()
+  %cst16k: int = prim::Constant[value=16384]()
+  %size : int[] = prim::ListConstruct(%cst768, %cst16k) 
+// CHECK: Half(*, *) = aten::view
```

Comment:
why it's unable to get shape of aten::view ? since `%p1` has known shape and `%size `is constant ...